src/language/Execution.jl

```diff
@@ -38,21 +38,23 @@ global storeEliminated
 global handleImpulses 
 
 function setOptions(options) 
-  global storeEliminated = true
-  if haskey(options, :storeEliminated)
-    global storeEliminated = options[:storeEliminated]
-    @show storeEliminated
-  end
-  global handleImpulses = false
-  if haskey(options, :handleImpulses)
-    global handleImpulses = options[:handleImpulses]
-    @show handleImpulses
-  end
-  global logTiming = false
-  if haskey(options, :logTiming)
-    global logTiming = options[:logTiming]
-    @show logTiming
-  end
+    global storeEliminated = true
+    if haskey(options, :storeEliminated)
+      global storeEliminated = options[:storeEliminated]
```

Comment:
This PR still leaves 2-space indents in several places. If it's been decided to move to 4-space indents, I suggest using [DocumentFormat.jl](https://github.com/ZacLN/DocumentFormat.jl) as described in #20. It's easy to reformat the whole package. Here's the result:

https://github.com/tshort/Modia.jl/tree/documentformat

I didn't submit that as a PR because I wasn't sure if this was decided. (I can if someone wants.)